### ISSUE TITLE
JetSmartFilters v1.6.0 Beta

### ISSUE THREAD

**Issue by MjHead** [Crocoblock/suggestions#737]
Hi,

We are almost getting ready to release JetSmartFilters v1.6.0 and we would like to ask you to help us test JetSmartFilters v1.6.0 Beta.

You can download beta version from your Crocoblock account.

## Hierarchical Filters. 
Now with Select filter type, you can create a structure based on one or several Taxonomies. Options for each next level of the hierarchy will be available only after selecting the previous one. Try it both with WooCommerce attributes and Custom Taxonomies:

### How it Works: 

For now, it works only with the Select type filter and only Taxonomy can be selected as a source. In the future, we’re going to expand the functionality. Moreover, it will be possible to build a hierarchy on the post relations basis. 

Create a new filter, choose the Select type, after which the “Is Hierarchical” switch will appear. If you check this switch, a control will pop up to create the hierarchy structure. 

From top to bottom, each element of this control is a new level of the hierarchy. It can be built both based on one Taxonomy as well as on combinations of several ones. If it is based on one Taxonomy, then when selecting the top-level option, the next level will display the child terms of the selected one. If the hierarchy consists of several Taxonomies, then on each next level, only elements that correspond to the combination of previously selected ones will be available. 

In case if you use WooCommerce, the leading convenience of this approach is that the attributes of products in WooCommerce are taxonomies; it means that with this functionality, it will be possible to do hierarchical filters by WooCommerce attributes.

Hierarchical filters can be combined with regular filters, but you can not combine two or more different hierarchical filters when filtering a single listing.

## Redirect on Filter Results Page
This new option allows creating a separate search form in which the user can apply the necessary filters, click “Search” and go to the page with the search results, where the filtered information will be displayed.   

### How it Works: 

You can create Filters and display them on the page in the same way as a standard workflow. The main point is that if you want all filters to work this way you need to select the triggering type by clicking on the button. 

It can be done with the help of the Apply Button widget (we decided not to add it globally yet). So, you need to create two pages: the first one should contain the filters and the second one will be showcase filter results.

You will need to enable the Apply Redirect option and indicate which page you want to redirect to (actually insert a page URL): https://prnt.sc/pnzacs 

On the page where you want to display the results, the widget of the required provider should be displayed - JetEngine, Elementor Pro, etc. When redirecting to this page, the filtering automatically works according to the parameters that were selected on the original page with the filters. Active filters also will be checked automatically (if the results page contains filters similar to the original page).

## Notes
For bugs and remarks, please reply with a comment on this issue here on GitHub.

Please keep this related to the Current beta Only! any other issue or feature request, feel free to open a new issue.

Many thanks for your support and help!

**Comment by Digital03** [comment]
Hi.

I have spanish site, with dates localized into my language. All works ok in PC also mobile views, but when apply filters on mobile device (chrome android) my dates change to english, also with pagination widget.


URL query with pagination reload page active:
?jet-smart-filters=jet-engine/listadoeventos&jet_paged=1&jet_paged=1

Duplicated $jet_paged.
Suggestion: more clean urls.

**Comment by Digital03** [comment]
Suggestion: Calendar view filter, to change ajax query with only push date.

**Comment by Digital03** [comment]
Sorry, im testing and the language issue was caused by other plugin conflict.

**Comment by MagicaDaGipnoz** [comment]
Hi, a really long waited feature. 
Hope that it will be [an updated tutorial](https://www.youtube.com/watch?v=F00H7xn8PF4) on youtube with current new features and current limitations. Thanks

**Comment by realstephenzhao** [comment]
Hi, when will filter bar(with Choose All option and responsive on mobile) be released?

**Comment by Gewerbe** [comment]
> Hallo.
> 
> Ich habe eine spanische Site mit Daten, die in meine Sprache lokalisiert sind. Alles funktioniert ok in PC auch in mobilen Ansichten, aber wenn Filter AJAX auf mobilen Geräten (Chrome Android) angewendet werden, ändern sich meine Daten in Englisch, auch mit Paginierungs-Widget.
> 
> URL-Abfrage mit aktiver Paginierungs
> -Neuladeseite : ?jet-smart-filters=jet-engine/listadoeventos&jet_paged=1&jet_paged=1
> 
> Dupliziertes $jet_paged.
> Vorschlag: mehr saubere URLs.



> Hallo, wann wird die Filterleiste (mit der Option Alle auswählen und auf Mobilgeräten reagierend) veröffentlicht?

Hallo Liebes Team, der Filter läst sich bei Android Handy nicht Öffnen. kennt sich jemand damit aus.
